dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -581,6 +585,8 @@ GENERAL_ASSET=asset
 
 MSG_INVALID_FOR_OS=The feature resource {0} is invalid for the current OS and will not be installed.
 
+MSG_SERVER_NEW_FEATURES_NOT_REQUIRED=The server does not require any additional features.  No features were installed.
```

Comment:
Update so there's only one space between the two sentences in "The server does not require any additional features. No features were installed."